frontend/simple/js/state.js

```diff
@@ -99,19 +117,45 @@ const getters = {
   unreadMessageCount (state) {
     let mailboxContract = store.getters.mailboxContract
     return mailboxContract && mailboxContract.messages.filter((msg) => !msg.read).length
+  },
+  // Logged In user's identity contract
+  identity (state) {
```

Comment:
`identity` is about as vague as it gets. Whose identity? What about their identity? Either this returns the identity contract Id, or the identity contract, either way it should be named after one or the other, so in this case it seems `currentUserContract` might work best.